inbc-program/inbc/parser.py

```diff
@@ -299,7 +305,8 @@ def sota(args) -> str:
                        "username",
                        "password",
                        "release_date",
-                       "path"
+                       "path",
+                       "deviceReboot"
```

Comment:
Why not just reboot instead of deviceReboot?  I think the device is implied and we can just say reboot.

Reply:
decided in IM to leave as deviceReboot